### PR TITLE
MfgSpecificPing command has been removed by latest ZAP submodule update

### PR DESCRIPTION
**Pull request by vivien-apple**
#### Problem
The latest submodules update from #4826 has broken the current `.zap` configuration. The manufacturer specific command `MfgSpecificPing` is gone.
As a result trying to regenerate the configuration with `./scripts/tool/zap_regen_all.py` results into generated content without this command.

 #### Summary of changes
* Add back the command to `examples/chip-tool/chip-tool.zap`
* Add back the command to `src/darwin/Framework/CHIP/chip-tool.zap`
* Add back the command to `examples/all-clusters-app/all-clusters-common/all-clusters-app.zap`

### CODE REVIEW

**Review by lazarkov — APPROVED** [review]

**Review by mspang — APPROVED** [review]

**Review by saurabhst — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

File: examples/bridge-app/bridge-common/gen/call-command-handler.cpp [hunk 1]
```diff
@@ -338,10 +338,61 @@ EmberAfStatus emberAfOnOffClusterServerCommandParse(EmberAfClusterCommand * cmd)
             wasHandled = emberAfOnOffClusterOffCallback();
             break;
         }
+        case ZCL_OFF_WITH_EFFECT_COMMAND_ID: {
```
Comment (bzbarsky-apple) [inline examples/bridge-app/bridge-common/gen/call-command-handler.cpp:341]: Why these changes here?

**Review by bzbarsky-apple — APPROVED** [review]

File: examples/bridge-app/bridge-common/gen/call-command-handler.cpp (hunk 1 above)
Comment (vivien-apple) [reply examples/bridge-app/bridge-common/gen/call-command-handler.cpp:341]: Those changes are not intentional. 

This is yet an other side effect of the submodule update. I will turn them back off.
